app/assets/stylesheets/application.scss

```diff
@@ -25,29 +25,36 @@
 
 .nav-tabs {
```

Comment:
you should be able to tack on `.tt-nav-tabs` here without breaking into another class below.

`align-items` and `justify-content` properties only apply when `display: flex` is set